Jahsie/template/main.scss

```diff
@@ -0,0 +1,182 @@
+/* CSS for GUI...*/
+@import url(http://fonts.googleapis.com/css?family=Lato:400,900);
+
+
+
+
+html {
+	font-size: 12px;
+}
+
+@media screen and (min-width: 900px){
+  body {
+    font-size: 16px;
+  }
+}
+@media screen and (min-width: 1200px){
+  html{
+    font-size: 18px;
+  }
+}
+
+body {
+	font-family: 'Lato', sans-serif;
+	color: grey;
+	p {
+		text-align: center;
+		font-size: .85rem;
+	}
+}
+
+#mainer {
+
+}
+
+#main {
+/*	width: 90%;
+	height: 90%;
+	overflow: auto;
+	margin: auto;
+	position: absolute;
+	top: 0; left: 0; bottom: 0; right: 0;
+*/
+  	/*top: 50%;
+  	-webkit-transform: translateY(-50%);
+  	-ms-transform: translateY(-50%);
+  	transform: translateY(-50%);*/
+}
+
+h1 {
+	font-weight: 900;	
+	text-align: center;
+}
+
+#intro {
+	position: relative;
+	box-sizing: border-box;
+	display: inline-block;
+	width: 20%;
+	float: left;
+	text-align: center;
+	padding: 3% 0px 0px 3%;
+	height: 100%;
+}
+
+.guts {
+	position: absolute;
```

Comment:
I haven't worked with scss before, so I didn't try this out in the code, but  Chrome's developer tools let you change css on the fly. Removing this declaration keeps the text where I think you want it to be.